baseline-network-linux-hld.md

```diff
@@ -0,0 +1,162 @@
+# Unified Open Networking Hardware Support
+
+[[_TOC_]]
+
+## Scope
+
+The current open network operating system (NOS) market is fragmented and has
+multiple competing projects with different focus, architecture and level of
+maturity. Despite those differences, all of the currently available,
+open network operating systems are utilizing a mainline Linux kernel and
+need to build and maintain platform-specific abstractions (e.g., out-of-tree
+drivers) for peripheral switch hardware.
+
+This proposal aims to reduce the fragmentation by providing a common base for
+networking hardware support, without addressing the question of how to abstract
+and control the switch forwarding plane itself.
+
+The intention of this project is not to replace or compete with any of the
+existing network operating systems, but rather to provide a common and
+extensible layer on which they can be built. Its principles and paradigms are
+largely aligned with and adapted from the
+[OCP OpenNetworkLinux Project](https://github.com/opencomputeproject/OpenNetworkLinux) (which has been moved to maintenance mode, pending archival).
+This proposal specifically seeks an agreement between the developers and
+maintainers of the aforementioned network operating system communities, to be
+able to share common implementations to support open networking hardware.
+
+Support for open networking hardware platforms is currently distributed over
+various different projects. Many platforms, especially older ones, are supported
```

Comment:
This is confusing. First there are many platforms using ONL [platform as in HW?]. Then there is SONiC [platform means the OS]. And finally DentOS [it also uses ONL but but it is a different platform?]

Reply:
I will try to write a more detailed definition for "platform". In general, i think the idea is to talk about "platform" == "whitebox" == "open switch hardware" here.